Source/CCZ4/TraceARemoval.hpp

```diff
@@ -17,21 +17,22 @@
 class TraceARemoval
 {
   public:
-    template <class data_t> struct Vars
+    struct Vars
     {
-        Tensor<2, data_t> h;
-        Tensor<2, data_t> A;
+        Tensor<2, amrex::Real> h;
+        Tensor<2, amrex::Real> A;
 
         template <typename mapping_function_t>
         AMREX_GPU_HOST_DEVICE void
         enum_mapping(mapping_function_t mapping_function);
     };
 
-    template <class data_t>
     AMREX_GPU_HOST_DEVICE void
-    operator()(const amrex::CellData<data_t> &cell) const
+    operator()(const amrex::CellData<amrex::Real> &cell) const
     {
-        auto vars = load_vars<Vars>(cell);
+        // NOLINTNEXTLINE [cppcoreguidelines-pro-type-member-init]
+        Vars vars;
+        load_vars(cell, vars);
```

Comment:
Does the following work?
```suggestion
        Vars vars = load_vars<Vars>(cell);
```

Reply:
No it doesn't. I will get rid of it in the next update anyway...

Reply:
Ok I fixed it.